op-e2e/sequencer_failover_test.go

```diff
@@ -8,7 +9,7 @@ import (
 
 // [Category: Initial Setup]
 // In this test, we test that we can successfully setup a working cluster.
-func TestSequencerFailover_SetupCluster(t *testing.T) {
+func xTestSequencerFailover_SetupCluster(t *testing.T) {
```

Comment:
no `t.Skip(string)`?

Reply:
Ahh that's a better idea! Forgot about this API.

Reply:
Same, updated the code and please help enqueue again, thank you!